### PR TITLE
releases: Increase to 14 profiles/parallel builds

### LINKED ISSUE #8954: More than 6 profiles?
Until now, I'm using two devices so I can get the desired number of profile. Is there a way to quickly add more than the extra 5 profiles?

I remember there was a way to unpack and pack the apk back again with some subtle changes and I once did this but can't remember how.

### PR DESCRIPTION
**Pull request by david-allison-1**
## Purpose / Description
A user requested 12

## Fixes
Fixes #8954

## Approach
Let's future proof it to 14 profiles (and avoid 13)

## How Has This Been Tested?

Untested

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by david-allison-1** [comment]
Main question would be whether this adds too much time to the release process

**Comment by mikehardy** [comment]
I'd rather not increase them permanently actually - each new file is 34MB or so pushed and stored on github and it starts to be an unreasonable use of their storage resources as we do publish quite frequently. That's the reason I only publish 2 parallels for alphas actually, with 5 for main releases

This is getting to be extremely niche I think, and I want to support it but not as a general case

I think it would be better to make the publish-parallel-release script take an argument of the number to build so users can actually drive it?

**Comment by mikehardy** [comment]
gradle caching means the extra builds don't take a lot of time, but they do take a lot of space still

File: tools/parallel-package-release.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ fi
 # Get on to the tag requested
 #git checkout $TAG
 
-BUILDNAMES='A B C D E'
+BUILDNAMES='A B C D E F G H I J K L M'
```
Comment (mikehardy) [inline tools/parallel-package-release.sh:32]: Maybe parameterize it by count, and then for a select few versions (like 2.15.1 since it was just requested, and whatever the final 2.15.x is) I can crank out a bigger batch

**Comment by david-allison-1** [comment]
TL;DR: Voting to close, if you feel the final paragraph is reasonable

I've done a silly here (wrong script). I'm not going to rebase the change to work on the correct script as I don't think it will change the outcome.

I only intended for this to affect the 'PROD' build, not the 'alpha/beta' builds. We definitely want to keep the regular releases small.

I don't think there's much value in spending a long time on this, but we risk onboarding issues with non-technical users (I don't think it works on Windows for example).

Do you feel it's feasible to quickly get this running on GitHub actions with a parameter? 

That feels like a compromise: keep it at 5 for releases, and allow users to trigger builds with an arbitrary number of files on their fork (where artefacts expire)

**Comment by mikehardy** [comment]
That's an idea with potential! We know it builds fine from scratch like that. Only thing to handle is creating a dummy keystore for them. For that just uploading the default keystore that Android Studio generates is fine - that could even go in source control with the default passwords for debug keystores and everything. Nothing needs to be secret at that point, and you can just use the github "artefacts" action to attach builds ?
